operator/v1/types_klusterlet.go

```diff
@@ -193,7 +193,8 @@ type AwsIrsa struct {
 	// The arn of the hub cluster (ie: an EKS cluster). This will be required to pass information to hub, which hub will use to create IAM identities for this klusterlet.
 	// Example - arn:eks:us-west-2:12345678910:cluster/hub-cluster1.
 	// +required
-	HubClusterArn string `json:"hubClusterArn"`
+	HubClusterArn     string `json:"hubClusterArn"`
+	ManagedClusterArn string `json:"managedClusterArn"`
```

Comment:
Could you please add a comment to describe what this field is for?

Reply:
if this is the required field, you would want a validation for minLength to be 1? Same for HubClusterArn